src/App.js

```diff
@@ -41,7 +41,7 @@ function App() {
     setIsFormSubmitted(true);
     setTimeout(function () {
```

Comment:
You could use a arrow function here: `setTimeout(() => setFormSubmitted(false))`